docs/CI_CD_GUIDE.md

```
@@ -0,0 +1,563 @@
+  - 💡 Suggestions
+  - ❓ Questions
+  - 💬 Other
+- Posts/updates a formatted table comment on the PR
+- Shows file locations and line numbers
+- Links to each comment
+- Provides statistics
+
+**Status**: ✅ NEW - Ready to use
+
+**Example Output**:
+```markdown
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)